addon/components/frost-bunsen-input-select/component.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash'
+import Ember from 'ember'
+import computed, {readOnly} from 'ember-computed-decorators'
+
+import InputMixin from 'ember-frost-bunsen/mixins/input'
+import ListInputMixin from 'ember-frost-bunsen/mixins/list-input'
+import layout from './template'
+
+export default Ember.Component.extend(InputMixin, ListInputMixin, {
```

Comment:
See how much easier that is to reason about? I no longer have to ask myself what crazy magic is being done in some other module, and if I have to refactor my code, I just have to look up at these five methods to know what I have to do. The burden placed on me as a developer to know what hooks are in this module has been wiped away!